SSSpinnerButton/SpinnerButton/SSSpinnerButton.swift

```diff
@@ -52,13 +53,22 @@ open class SSSpinnerButton: UIButton {
             layer.cornerRadius = newValue
         }
     }
+    @IBInspectable var setRippleEffect: Bool = false {
+        didSet {
+            if setRippleEffect {
+                self.setRippleEffect(rippleEffectAnimationDuration: 0.3, initialOpacity: 0.5, rippleEffectColor: .white, rippleEffectPercent: 0.45, initalRippleEffectPercent: 0.3)
```

Comment:
@buddhdev-jay can we set the ripple effect colour dynamic instead of fixed white?